api/v1beta1/disruption_types.go

```diff
@@ -43,11 +43,12 @@ type DisruptionSpec struct {
 	// +nullable
 	AdvancedSelector []metav1.LabelSelectorRequirement `json:"advancedSelector,omitempty"` // advanced label selector
 	// +nullable
-	Filter          *DisruptionFilter `json:"filter,omitempty"`
-	DryRun          bool              `json:"dryRun,omitempty"`          // enable dry-run mode
-	OnInit          bool              `json:"onInit,omitempty"`          // enable disruption on init
-	Unsafemode      *UnsafemodeSpec   `json:"unsafeMode,omitempty"`      // unsafemode spec used to turn off safemode safety nets
-	StaticTargeting bool              `json:"staticTargeting,omitempty"` // enable dynamic targeting and cluster observation
+	Filter               *DisruptionFilter `json:"filter,omitempty"`
+	DryRun               bool              `json:"dryRun,omitempty"`          // enable dry-run mode
+	OnInit               bool              `json:"onInit,omitempty"`          // enable disruption on init
+	Unsafemode           *UnsafemodeSpec   `json:"unsafeMode,omitempty"`      // unsafemode spec used to turn off safemode safety nets
+	SpecifiedEnvironment string            `json:"specifiedEnv,omitempty"`    // optional field limiting where disruptions can ru
```

Comment:
typo - can run (at the end of the sentence)

Reply:
does `Environment`  clash with some thing else?

Reply:
Maybe we should be more specific in the comment and say something like: `limiting in which kubernetes cluster disruptions can run`

Reply:
What do you think of `AllowedEnvironment` || `AllowedEnv` or `LimitedEnvironment` ||`LimitedEnv`?

Reply:
I'd also just go with `Environment` to keep it as simple as possible.